### ISSUE TITLE
Lesson contribution - instructor checkout – Navigating Files and Directories

### ISSUE THREAD

**Issue by ZoeYang2020** [datacarpentry/shell-genomics#301]
For the Introduction to the Command Line for Genomics, Navigating Files and Directories

I tried to use the material as the teaching demo, I found it will be better if we have the  tree structures to show learner the organizations of the files and directory at the begin of teaching. 
For example, we can  move the the figures for Relative path resolution 
https://datacarpentry.org/shell-genomics/fig/filesystem-challenge.svg to the beginning of the lesson.  Introduce the learner how the files and directory organized, then the learner will have the basic ideas where they are, then it will be easier for them to follow the commands.

**Comment by akshayparopkari** [comment]
Hi @ZoeYang2020, thanks for your comment!

I agree that introducing the tree structure of the file system is a powerful visual for learners to begin understanding the filesystem. Previously, I have successfully used a toy example diagram from SWC Unix lesson provided here - https://swcarpentry.github.io/shell-novice/figures/index.html. I find that this figure is simple and helps with transitioning into navigating the `shell_data` folder. 

I am curious to hearing other ideas on this topic. Would we clarify filesystem visual via restructuring the lesson content or invite the instructors to utilize current resources from other Carpentries lessons. What do you think, @datacarpentry/shell-genomics-maintainers?

**Comment by aschuerch** [comment]
Thank you @ZoeYang2020 for this suggestion to improve the lesson. I think how the concept of the hierarchical filesystem is introduced can vary quite a bit depending on the background and level of your learners ('meeting learners where they are' ) but also your own preference. Personally, I often use a whiteboard to draw the growing structure of the filesystem as we move along in the lesson. This gives the learner a visual aid that is independent from what I show on the computer screen or their own screen. 
These different options are something that might be useful for other instructors that are preparing to teach this lesson. I would suggest to add a paragraph to the Instructor Notes at https://datacarpentry.org/shell-genomics/guide/index.html .

**Comment by akshayparopkari** [comment]
Thanks, @aschuerch! I agree, adding a paragraph to Instructor Notes would be quite useful.

@ZoeYang2020, please feel free to submit a PR with updated Instructor Notes. For step-by-step guide on contributing to Carpentries lesson content, please go through [instructions for Carpentries lesson contribution](https://github.com/dmgt/swc_github_flow/blob/master/for_novice_contributors.md#instructions-for-contributing-to-the-carpentries-lesson-materials-using-graphical-or-command-line-interfaces-with-git-and-github).

**Comment by esallychang** [comment]
Yes, agreed with all the comments above. @ZoeYang2020 we would LOVE an edit to add details to the instructor notes about this if you are at all interested. If you don't have time to get familiar with GitHub but want to type up some text and send it to me, I am happy to do the pull request and edit the file itself to add your suggestions.
